tools/load-examples.groovy

```diff
@@ -112,168 +112,177 @@ class Main {
 		Main.class.getClass().getResource(new File(scheduler_home, "dist/lib/httpclient-4.5.2.jar").absolutePath);
 		def boundary = "---------------" + UUID.randomUUID().toString();
 
-		// For each example directory
-		example_dir.eachDir() { dir ->
+		// Retrieve the ordered busket list
+		def ordered_bucket_list_file = new File(examples_dir, "ordered_bucket_list")
+		ordered_bucket_list_file.text.split(",").each { bucket_name ->
 
-			def metadata_file = new File(dir.absolutePath, "METADATA.json")
-			if (metadata_file.exists())
+			// If the bucket directory exists
+			def bucket_dir = new File(examples_dir, bucket_name)
```

Comment:
Don't you want to use a file extension for the new File? bucket_name + ".json" maybe?

Reply:
Or other extension in fact

Reply:
I had in mind that , but this workaround (provide an ordered bucket list to follow for the deployment) must be removed as soon as possible, since anyone can create any bucket at anytime.